StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Link/LinkController.swift

```diff
@@ -36,6 +36,15 @@ import UIKit
         case canceled
     }
 
+    @frozen @_spi(STP) public enum AuthorizeResult {
```

Comment:
Nit to be aligned with `VerificationResult`?

```suggestion
    @frozen @_spi(STP) public enum AuthorizationResult {
```